Source/PeleLMeX_Plot.cpp

```diff
@@ -116,6 +120,10 @@ PeleLM::WritePlotFile()
   ncomp += 1;
 #endif
 
+#ifdef PELELM_USE_RAD
+  ncomp += 3;
```

Comment:
also need to guard with `do_rad_solve` ?

Reply:
There are two control flags for the radiation module, or any other PELEMP modules in general.
One is at compile time, and one is at runtime through the 'do_rad_solve' flag.

I considered 'ncomp +=3' belongs to the compile time, but it is fine if you think it fits to runtime.

Reply:
I do believe that the runtime guard should be used here. Otherwise, if you compile but don't run with radiation, the multifab for plot variables will have 3 empty, unnamed components. 

It is pretty messy and bug-prone how we have 3 separate places in the code for counting pltvars, adding their names, and copying the data, and everything has to be in the same order for things to work. We should refactor this at some point in the future.